_posts/2024-05-18-developing-openwrt-in-qemu.adoc

```diff
@@ -0,0 +1,88 @@
+---
+title: Developing OpenWRT in QEMU with Eclipse Che
+layout: post
+author: Oleksii Orel
+description: >-
+  Developing OpenWRT helloworld package in QEMU machine emulator and virtualizer with Eclipse Che.
+categories: []
+keywords: ['openwrt', 'quemu', 'gdb', 'quemu', 'developing ', 'workspace']
+slug: /@olexii.orel/openwrt-helloworld-package
+---
+
+OpenWrt is a highly extensible GNU/Linux distribution for embedded devices (typically wireless routers). OpenWrt is built from the ground up to be a full-featured, easily modifiable operating system for embedded devices. You can have all the features you need with no bloat, powered by a modern Linux kernel.
+
+We can create and develop OpenWRT CMake packages with Eclipse Che.
+
+In this blog post, we will review developing OpenWRT in QEMU with Eclipse Che in 4 parts:
+
+<<part1>>
+
+<<part2>>
+
+<<part3>>
+
+<<part4>>
+
+
+NOTE: To help illustrate this blog post we will use an https://github.com/che-incubator/openwrt-helloworld-package.git[OpenWrt HelloWorld package].
```

Comment:
```suggestion
NOTE: In this blog post, we will use https://github.com/che-incubator/openwrt-helloworld-package.git[OpenWrt HelloWorld package].
```

Reply:
I would probably duplicate devfile in the blog-post

Reply:
I have fixed it.